README.md

```diff
@@ -880,7 +892,7 @@ constraint_list = ConstraintList()
 constraint_list.add(constraint)
 ```
 
-### Class: ConstraintFcn
+### Class: ConstraintFcn (pour TRACK_STATE, TRACK_MARKERS ect donner quelques exemples de parametres, expliciter)
```

Comment:
Is this intended?

Reply:
On souhaitait rajouter tous les paramètres que l'utilisateur peut rajouter tels que "min_bound" "phase"